pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -332,11 +332,6 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 				// ReplicationManager.syncReplicationController() -> updateReplicaCount()
 				{
```

Comment:
I've looked through RC controller, especially when writing job controller,
and I didn't find a place where it's being used. All the retrieval is done
through list and watch. Similarly jobs don't have get nor update.
On Oct 23, 2015 10:50 PM, "Jordan Liggitt" <notifications@github.com> wrote:

> In pkg/cmd/server/bootstrappolicy/policy.go
> <https://github.com/openshift/origin/pull/5376#discussion_r42912856>:
>
> > @@ -332,11 +332,6 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
> >  				},
> >  				// ReplicationManager.syncReplicationController() -> updateReplicaCount()
> >  				{
>
> Are you sure we don't need get any more?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5376/files#r42912856>.
>